src/item.cpp

```diff
@@ -114,9 +94,9 @@ Item::Item(const rapidjson::Value &json) :
         enchanted_ = true;
 	
     if (json.HasMember("shaper") && json["shaper"].IsBool())
```

Comment:
Should have a check for the actual boolean value

Reply:
For future-proofing, ok. Probably also wants to assert that an item can only be Shaper or Elder, not have a latter value discard the former via the new type/enum.